latch_cli/docker_utils/__init__.py

```diff
@@ -202,9 +203,12 @@ def generate_dockerfile(pkg_root: Path, outfile: Path) -> None:
             print("  - base image:", config.base_image)
             print("  - latch version:", config.latch_version)
     except FileNotFoundError as e:
-        raise RuntimeError(
-            "Could not find a .latch/config file in the supplied directory. If your workflow was created prior to release 2.13.0, you may need to run `latch init` to generate a .latch/config file."
-        ) from e
+        print(
+            "Could not find a .latch/config file in the supplied directory. Recreating configuration."
+        )
+        create_and_write_config(latch_constants.base_image, pkg_root)
```

Comment:
sort of a nit but itd be cleaner to have `create_and_write_config` only take in a `pkg_root` and have the base image be optional, defaulting to the constant `latch_constants.base_image` if not provided

Reply:
Sounds good.